python/ppml/src/bigdl/ppml/fl/nn/pytorch/aggregator.py

```diff
@@ -102,27 +102,44 @@ def put_client_data(self, client_id, data):
 
 
     def aggregate(self):
-        input, target = [], None        
-        for ndarray_map in self.client_data.values():
+        input, target = [], None
+        # to record the order of tensors with client ID
+        for cid, ndarray_map in self.client_data.items():
             for k, v in ndarray_map.items():
                 if k == 'input':
-                    input.append(torch.from_numpy(v))
+                    input.append((cid, torch.from_numpy(v)))
```

Comment:
how's the client ID determined?

Reply:
I think I can make this be set in `ppml-conf.yaml` (if not set it would be random uuid). And the order will be default the the string comparison order. So just set `1, 2, ..` if this make sense.

Reply:
Maybe set it in `init_fl_context`?

Reply:
[new commit](https://github.com/intel-analytics/BigDL/pull/4699/commits/ecab9fe823d9e910e872cefb85064c89f33adf19) supports pass client_id in `FLContext.init_fl_context()` and changes the FLClient to singleton by default, the same as we previously implemented in Scala and Py4J version.

Reply:
This is changed into `Estimator.from_torch(client_id=...)`